src/core/tokenHandler.js

```diff
@@ -106,64 +107,58 @@ export default function tokenHandlerFactory(options) {
          * @param {String} newToken
          * @returns {Promise<Boolean>} - resolves true if successful
          */
-        setToken: function setToken(newToken) {
+        setToken(newToken) {
             return tokenStore.enqueue(newToken);
         },
 
         /**
          * Extracts tokens from the Client Config which should be received on every page load
          * @returns {Promise<Boolean>} - resolves true when completed
          */
-        getClientConfigTokens: function getClientConfigTokens() {
-            var self = this;
-            var clientTokens = _.map(module.config().tokens, function(serverToken) {
-                return {
-                    value: serverToken,
-                    receivedAt: Date.now()
-                };
-            });
+        getClientConfigTokens() {
+            const clientTokens = _.map(module.config().tokens, serverToken => ({
```

Comment:
How about Array.map() in place of _.map() ?
We may never be free of lodash but we can move in the direction of native features.

Reply:
Well, I would prefer to keep lodash here, as the data comes from the config and could not be a true array. Anyway, I will try to replace it as well.